test/integration/basic_nsmd_deploy_restart_test.go

```diff
@@ -69,6 +69,10 @@ func TestNSMgrRestartDeploy(t *testing.T) {
 	g.Expect(restarts).To(Equal(int32(1)))
 	g.Expect(<-errChan).To(BeNil())
 	g.Expect(<-prevLogsChan).To(ContainSubstring("SIGABRT: abort"))
+
+	logrus.Info("Waiting for k8s plugin to re-register in restarted NSMD...")
+	err = k8s.WaitLogsContainsRegex(nodesConf[0].Nsmd, "nsmd-k8s", "Successfully re-registered plugin", fastTimeout)
```

Comment:
please consider not use fast timeout here. Log api is slow in kubernetes and it could be a reason of test failing.

Reply:
- [ ] use defaultTimeout